client/src/components/CreateBoardModal/CreateBoardModal.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Lifecycle from './Lifecycle';
+import { Button, Dialog, Classes, InputGroup, Label } from '@blueprintjs/core';
+
+class CreateBoardModal extends React.Component {
+  constructor(props) {
```

Comment:
use class variables instead of constructor, you can also remove the bindings from the constructor using arrow functions, check this guide https://codeburst.io/use-class-properties-to-clean-up-your-classes-and-react-components-93185879f688